.github/workflows/release.yml

```
@@ -0,0 +1,146 @@
+                  unzip -l "$file" | head -15
+                fi
+                echo "---"
+              fi
+            done
+          else
+            echo "Warning: No dist directory found - check GoReleaser configuration"
+          fi
+
+      - name: Upload additional artifacts
+        if: always() && hashFiles('dist/**/*') != ''
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)